website/templates/emails/welcome_osf4i.html.mako

```diff
@@ -11,37 +11,50 @@
+Keep your research materials and data private, make it accessible to specific others with view-only links, or make it publicly accessible. You have full control of what parts of your research are public and what remains private. <a href="http://help.osf.io/m/projects/l/524048-control-your-privacy-settings/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Explore privacy settings.</a><br>
 <br>
+
 <h4>Register your research</h4>
-Create a permanent, time-stamped version of your projects and files.  Do this to preregister your design and analysis plan to conduct a confirmatory study, or archive your materials, data, and analysis scripts when publishing a report.<br>
+Create a permanent, time-stamped version of your projects and files.  Do this to preregister your design and analysis plan to conduct a confirmatory study, or archive your materials, data, and analysis scripts when publishing a report. <a href="http://help.osf.io/m/registrations/l/524205-register-your-project/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Read about registrations.</a><br>
 <br>
+
 <h4>Make your work citable</h4>
-Every project and file on the OSF has a permanent unique identifier, and every registration can be assigned a DOI.  Citations for public projects are generated automatically so that visitors can give you credit for your research.<br>
+Every project and file on the OSF has a permanent unique identifier, and every registration can be assigned a DOI.  Citations for public projects are generated automatically so that visitors can give you credit for your research. <a href="http://help.osf.io/m/sharing/l/524208-create-dois/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Learn more.</a><br>
 <br>
+
 <h4>Measure your impact</h4>
-You can monitor traffic to your public projects and downloads of your public files.<br>
+You can monitor traffic to your public projects and downloads of your public files. <a href="http://help.osf.io/m/projects/l/524052-view-analytics/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Discover analytics.</a><br>
 <br>
+
 <h4>Connect services that you use</h4>
-GitHub, Dropbox, Google Drive, Box, Dataverse, figshare, Amazon S3, ownCloud, Bitbucket, GitLab, OneDrive, Mendeley, Zotero.  Do you use any of these? Link the services that you use to your OSF projects so that all parts of your project are in one place.<br>
+OSF integrates with GitHub, Dropbox, Google Drive, Box, Dataverse, figshare, Amazon S3, ownCloud, Bitbucket, GitLab, OneDrive, Mendeley, and Zotero. Do you use any of these? Link the services that you use to your OSF projects so that all parts of your research project are in one place. <a href="http://help.osf.io/m/addons/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Learn about add-ons.</a><br>
```

Comment:
"OSF integrates with GitHub, Dropbox, Google Drive, Box, Dataverse, figshare, Amazon S3, ownCloud, Bitbucket, GitLab, OneDrive, Mendeley, and Zotero. Do you use any of these? Link the services that you use to your OSF projects so that all parts of your research project are in one place"
should be 
"OSF integrates with GitHub, Dropbox, Google Drive, Box, Dataverse, figshare, Amazon S3, ownCloud, Bitbucket, GitLab, OneDrive, Mendeley, and Zotero. Link the services that you use to your OSF projects so that all parts of your research are in one place"